safe-global/safe-wallet-web
# fix: port balance change

## What it solves

Porting the balance change to the new flow.

## How this PR fixes it

The balance change styles have been updated according to the new design.

## How to test it

Create a transaction on a Redefine-supported chain and observe the new balance changes that match [the design](https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?type=design&node-id=5929-177175&mode=design&t=3yvXPV6A6sQS77QC-0).

## Screenshots

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/bb22c8ee-b259-42d3-8144-c60a3e87bb99)

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/1c30b667-6616-4673-83e8-3c9d4562f441)

## Checklist
